mungegithub/issue-labeler/OWNERS

```diff
@@ -1,4 +1,4 @@
-assignees
+assignees:
```

Comment:
you mean separate PR or just a separate commit? It was mostly for testing purposes